06.wc/lib/wc.rb

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  line_opt = false
+
+  opt = OptionParser.new
+  opt.on('-l') { line_opt = true }
+
+  opt.parse!(ARGV)
+
+  # ARGFでファイルからの入力かパイプからの入力かを判断
+  # https://docs.ruby-lang.org/ja/latest/class/ARGF.html
+  if ARGF.argv.count.zero?
+    # 標準入力
+    wc = Wc.new(ARGF.gets(nil))
+    puts format_wc(wc.count_table, line_opt)
```

Comment:
```suggestion
    output_stdin(line_opt)
```

みたいになってた方がif/elseで対称性があって美しいかも。（好みの問題ですが）